### PR TITLE
RegNum search results

### PR DESCRIPTION
**Pull request by bryan-gilbert**
Provide a component to show the financing statements that are retrieved via a registration number search.  This is, purposely, meant to be a code like display as we are developing while UX research is underway.

### CODE REVIEW

File: ppr-ui/src/search/SearchResultsRegistrationNumber.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    </p>
+    <div>
+      <pre>{{ results }}</pre>
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import {computed, createComponent} from "@vue/composition-api"
+import {Data} from "@vue/composition-api/dist/ts-api/component"
```
Comment (WalterMoar) [inline ppr-ui/src/search/SearchResultsRegistrationNumber.vue]: Your VS Code still isn't formatting code. Also, we're using single quotes in TypeScript.

File: ppr-ui/src/search/SearchResultsRegistrationNumber.vue [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<template>
```
Comment (WalterMoar) [inline ppr-ui/src/search/SearchResultsRegistrationNumber.vue]: I don't think we want a component for the results from every type of search. I know this is temporary, but it should at least be leaning in the direction of being a reusable component.

File: ppr-ui/src/search/SearchResultsRegistrationNumber.vue (hunk 1 above)
Comment (bryan-gilbert) [reply ppr-ui/src/search/SearchResultsRegistrationNumber.vue]: Help.  I used VS code this time.

File: ppr-ui/src/search/SearchResultsRegistrationNumber.vue (hunk 2 above)
Comment (WalterMoar) [reply ppr-ui/src/search/SearchResultsRegistrationNumber.vue]: ^^^ Worst, vague, rushed, comment ever.

Comment (bryan-gilbert) [reply ppr-ui/src/search/SearchResultsRegistrationNumber.vue]: but thankfully we can chat ... hope the new commit helps!

**Review by WalterMoar — APPROVED** [review]

**Pull request merged**